testcase_maker/generator.py

```diff
@@ -15,6 +15,8 @@
 if TYPE_CHECKING:
     from testcase_maker.values import ValueGroup
 
+import logging 
+logging.basicConfig(filename='generator.log', level=logging.DEBUG, format='%(asctime)s %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p')
```

Comment:
A few things ids prefer for the logging.

1. I think the default level is better to be INFO.
2. Showing log to stdout should be good enough, don't need to save to file.

Reply:
Alright, ill make the changes.